recipes/rosbuild/meta.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: ros-rosbuild
+  version: 1.14.6
+
+source:
+  url: https://github.com/ros-gbp/ros-release/archive/release/melodic/rosbuild/1.14.6-0.tar.gz
+  fn: ros-rosbuild-1.14.6.tar.gz
+  sha256: 3c95011a3c521dc30e8b7f4399fdc46f0e3702978b9060f309529e6e8f396e43
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+  host:
+    - ros-conda-mutex * melodic
+    - ros-conda-base
+    - catkin_pkg
+    - pkg-config
+    - ros-catkin
+  run:
+    - ros-conda-mutex * melodic
+    - ros-conda-base
+    - ros-catkin
+    - ros-message-runtime
+    - ros-message-generation
+
+test:
+  requires:
+    - rospkg
+  commands:
+    - export ROS_ROOT=$PREFIX && python -c "import rospkg;s = rospkg.RosPack();ret = s.get_path('rosbuild');print(ret)"
```

Comment:
Should you add an activate and deactivate script to set ``ROS_ROOT`` automatically?
so when someone installs this package on their environment, this will be set transparently

Reply:
You can see one example here
https://github.com/conda-forge/gdal-feedstock/tree/master/recipe/scripts

Reply:
yeah, indeed i have such an activation script in the `ros-catkin` package. It's necessary to correctly set the environment variables for the catkin workspace, and it will probably also set the ROS_ROOT.
In order to make testing leaner, I didn't use that package here.